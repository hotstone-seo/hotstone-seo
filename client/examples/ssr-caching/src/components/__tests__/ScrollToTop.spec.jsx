import React from 'react';
import ScrollToTop from '../ScrollToTop.jsx';

test('Should render ScrollToTop correctly', () => {
  const tree = enzyme.mount(
    <ScrollToTop.WrappedComponent location="/">
      <div style={{ height: 500 }} />
    </ScrollToTop.WrappedComponent>
  );

  tree.setProps({
    location: '/test'
  });
  tree.setProps({
    location: '/test2'
  });
  expect(tree).toMatchSnapshot();
});
