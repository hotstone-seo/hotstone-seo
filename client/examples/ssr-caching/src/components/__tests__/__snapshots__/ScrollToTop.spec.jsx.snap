// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ScrollToTop correctly 1`] = `
<ScrollToTop
  location="/test2"
>
  <div
    style={
      Object {
        "height": 500,
      }
    }
  />
</ScrollToTop>
`;
